chore(deps): lock file maintenance (#305)
api3dao/api3-docs: docs/airnode/next/reference/admin-cli-commands.md

<divider/>

<!-- prettier-ignore-start -->
<Fix>Be consistent on how we write the terminology (pick one of "sponsor wallet", "sponsorWallet" and "Sponsor Wallet")</Fix>
<Fix>Link "sponsor wallet", "sponsor address", "sponsor", ... to the corresponding sections in the docs</Fix>
<Fix>Use "`argumentName` (optional)" instead of putting the "(optional)" in backtics with the argument name</Fix>
<Fix>Go through this link and reference more API3 terms (link them to docs)</Fix>
<!-- prettier-ignore-end -->

### `sponsor-requester`

> [Sponsors](..concepts/sponsorship.md) a requester contract so that its requests can be fulfilled by the sponsorWallet of

Reviewer: ```suggestion
[Sponsors](../concepts/sponsor.md) a requester contract so that its requests can be fulfilled by the sponsorWallet of
```

### `request-withdrawal`

> Requests a [withdrawal](./concepts/sponsor-wallet.html#withdrawals) from a `sponsorWallet` managed by an Airnode and

Reviewer: ```suggestion
Requests a [withdrawal](../concepts/sponsor.md#withdrawals) from a `sponsorWallet` managed by an Airnode and
```

### `set-whitelist-expiration`

Called by a super admin to set the whitelisting expiration of a user for the Airnode–endpoint pair. This can hasten
expiration in the case the new expiration timestamp is prior to a previously set timestamp.

- `mnemonic`: Used to pay gas costs from the mnemonic's default account unless a `derivation-path` is specified. The
  mnemonic must be at least an Admin or Airnode wallet.
- `provider-url`: A valid cloud provider URL.

> - `endpoint-id`: The [`endpointId`](./reference/deployment-files/config-json.md#triggers) for which permission is

Reviewer: ```suggestion
- `endpoint-id`: The [`endpointId`](deployment-files/config-json.md#triggers) for which permission is
```

### `extend-whitelist-expiration`

Called by an admin to extend the whitelist expiration of a user for the Airnode–endpoint pair. This command expects that
the new expiration timestamp is later then the previously set timestamp.

- `mnemonic`: Used to pay on-chain gas cost for this command's transaction. The mnemonic must be at least an Admin or
  Airnode wallet.
- `provider-url`: A valid cloud provider URL.

> - `endpoint-id`: The [`endpointId`](./reference/deployment-files/config-json.md#triggers) for which permission is

Reviewer: ```suggestion
- `endpoint-id`: The [`endpointId`](deployment-files/config-json.md#triggers) for which permission is
```

### `set-whitelist-status-past-expiration`

Called by a super admin to set the whitelist status of a user past expiration for the Airnode–endpoint pair. This
command can be used to make whitelisting permanent in cases where it is needed to allow requests even after expiration
elapses.

- `mnemonic`: Used to pay on-chain gas cost for this command's transaction. The mnemonic must be at least an Admin or
  Airnode wallet. The default address of the mnemonic will be used unless a derivation-path is provided.
- `provider-url`: A valid cloud provider URL.

> - `endpoint-id`: The [`endpointId`](./reference/deployment-files/config-json.md#triggers) for which permission is

Reviewer: ```suggestion
- `endpoint-id`: The [`endpointId`](deployment-files/config-json.md#triggers) for which permission is
```

  --user-address 0x2c2e12... \
  --airnode-address 0xe1e0dd... \
```

<divider/>

### `user-is-whitelisted`

Called to check if a requester is whitelisted to use the Airnode–endpoint pair.

- `provider-url`: A valid cloud provider URL.
- `airnode-requester-rrp-authorizer`: The authorizer contract address.

> - `endpoint-id`: The [`endpointId`](./reference/deployment-files/config-json.md#triggers) for which permission is

Reviewer: ```suggestion
- `endpoint-id`: The [`endpointId`](deployment-files/config-json.md#triggers) for which permission is
```
